### ISSUE TITLE
brew doctor undefined method `/' for nil:NilClass

### ISSUE THREAD

**Issue by MiteshShah** [Homebrew/homebrew#50184]
# Please follow the general troubleshooting steps first:

- [X] Ran `brew update` and retried your prior step?
```
╭ Mitesh at Miteshs-MacBook-Pro in ~ using 16-03-17 - 10:25:51 
╰ ○ brew update                    
Already up-to-date.
╭ Mitesh at Miteshs-MacBook-Pro in ~ using 16-03-17 - 10:27:00 
╰ ○ brew update
Already up-to-date.
```
- [ ] Ran `brew doctor`, fixed as many issues as possible and retried your prior step?
```
╭ Mitesh at Miteshs-MacBook-Pro in ~ using 16-03-17 - 10:27:04 
╰ ○ brew doctor
Error: undefined method `/' for nil:NilClass
Please report this bug:
    https://git.io/brew-troubleshooting
/usr/local/Library/Homebrew/os/mac/xcode.rb:82:in `uncached_version'
/usr/local/Library/Homebrew/os/mac/xcode.rb:71:in `version'
/usr/local/Library/Homebrew/diagnostic.rb:419:in `check_for_osx_gcc_installer'
/usr/local/Library/Homebrew/cmd/doctor.rb:34:in `block in doctor'
/usr/local/Library/Homebrew/cmd/doctor.rb:27:in `each'
/usr/local/Library/Homebrew/cmd/doctor.rb:27:in `doctor'
/usr/local/Library/brew.rb:83:in `<main>'
```

brew doctor it self broken

**Comment by DomT4** [comment]
Can you provide a `brew config` and `cd /usr/local && git status`?

**Comment by MiteshShah** [comment]
```
╭ Mitesh at Miteshs-MacBook-Pro in ~ using 16-03-17 - 10:33:55 
╰ ○ brew config
HOMEBREW_VERSION: 0.9.5
ORIGIN: https://github.com/Homebrew/homebrew
HEAD: 1f9845a90a8608386e2dfae5a3148f9af6ea6f16
Last commit: 22 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit haswell
OS X: 10.11.3-x86_64
Xcode: N/A
Error: undefined method `/' for nil:NilClass
Please report this bug:
    https://git.io/brew-troubleshooting
/usr/local/Library/Homebrew/os/mac/xcode.rb:82:in `uncached_version'
/usr/local/Library/Homebrew/os/mac/xcode.rb:71:in `version'
/usr/local/Library/Homebrew/cmd/config.rb:43:in `clt'
/usr/local/Library/Homebrew/cmd/config.rb:154:in `dump_verbose_config'
/usr/local/Library/Homebrew/cmd/config.rb:7:in `config'
/usr/local/Library/brew.rb:83:in `<main>'
```

```
╭ Mitesh at Miteshs-MacBook-Pro in ~ using 16-03-17 - 10:33:57 
╰ ○ cd /usr/local && git status
On branch master
nothing to commit, working directory clean
```

**Comment by DomT4** [comment]
Looking into this, Thanks.

**Comment by tdsmith** [comment]
Fixed for now by c6183aa; thanks for reporting.

**Comment by MiteshShah** [comment]
Thanks everything working now :+1:
